go.mod

```diff
@@ -3,7 +3,7 @@ module knative.dev/func
 go 1.23
 
 // this is required bacause of bad dep in github.com/openshift-pipelines/pipelines-as-code
-replace k8s.io/client-go => k8s.io/client-go v0.29.3
+replace github.com/imdario/mergo => dario.cat/mergo v1.0.1
```

Comment:
Is the comment above still valid?

Reply:
Yes, it's used in and due to `pipeline-as-code`. 
ref: https://github.com/openshift-pipelines/pipelines-as-code/blob/v0.31.0/go.mod#L67